Packs/Office365AndAzureAuditLog/Integrations/MicrosoftPolicyAndComplianceAuditLog/README.md

```diff
@@ -21,6 +21,13 @@ Use the integration to get logs from the O365 service.
     3. Run the ***o365-auditlog-auth-test*** command to verify that the authorization process was implemented correctly.
 
 
+## Required Permissions To Search Audit Logs
+- Minimum Required Exchange roles are **Audit Logs** or **View-Only Audit Logs**
+- Go to [The Microsoft Admin Portal](https://admin.microsoft.com/Adminportal#/homepage)
+- Click **Show All** --> **Roles** --> **Roles Assignments** --> **Exchange section**.
+- Click **Add role group** --> Choose for name and description --> Choose Audit Logs or View-Only Audit Logs Roles --> Choose the members --> Add role group
```

Comment:
Aren't `Audit Logs` and `View-Only Audit Logs` roles that are options in the product? If so, let's bold them to make that clear.
```suggestion
- Click **Add role group** --> Choose for name and description --> Select the **Audit Logs** or **View-Only Audit Logs** roles --> Select the members to apply the role(s) to --> Click **Add role group**
```